Content.Server/Medical/BiomassReclaimer/BiomassReclaimerComponent.cs

```diff
@@ -49,6 +55,18 @@ public sealed partial class BiomassReclaimerComponent : Component
         [DataField, ViewVariables(VVAccess.ReadWrite)]
         public float YieldPerUnitMass = 0.4f;
 
+        /// <summary>
+        /// How many seconds to take to insert an entity per unit of its mass.
+        /// </summary>
+        [DataField, ViewVariables(VVAccess.ReadWrite)]
+        public float BaseInsertionDelay = 0.1f;
```

Comment:
uhh is this really that necesary? cant the delay be an arbitrary number?

Reply:
what do you mean? 0.1 is already an arbitrary number

Reply:
as in make it a static number instead of it scaling with mass

Reply:
i think scaling with mass is better because why should it take the same time to reclaim a cow as it takes to reclaim a piece of wheat

Reply:
I don't really see how the extra complexity attributes that much and it can add some odd cases and inconsistencies where something takes a LOT of time cuz sometimes mass is cursed. Tho i don't really care that much and if you think its pros are more than the cons feel free to resolve this one

Reply:
i think it's better this way since otherwise it'd either be too fast on large mobs or too slow on small mobs
if set to, say, 5 seconds, it'd be somewhat strong to be able to recycle entire cows in 5 seconds
and if set to, say, 15 seconds, it'd be extremely annoying to have to wait 15 seconds to recycle a single mouse
otherwise, yeah, i don't believe the negatives really matter here, so i'll resolve this